### PR TITLE
TK1.2: add service arguments to auth.threatgrid

### PR DESCRIPTION
**Pull request by frenchy64**
Related threatgrid/iroh#3986
Extracted from https://github.com/threatgrid/ctia/pull/924

Almost everything in `ctia.init` will be rewritten when we introduce Trapperkeeper--this adds some stubs in the meanwhile.

I'm using TK##.## to keep track of these PR's and their order. You can ignore them, or see the planned list [here](  https://github.com/threatgrid/ctia/pull/924#actualprs). If a PR is ready for review, assume you can review in any order.

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: TK1.2: add service arguments to auth.threatgrid
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

**Review by vemv — APPROVED** [review]
lgtm!

File: src/ctia/auth/threatgrid.clj [hunk 1]
```diff
@@ -70,8 +70,8 @@
                                                    (get default-capabilities))}))))
         auth/denied-identity-singleton)))
 
-(defn make-auth-service []
-  (let [{:keys [whoami-url cache]} (p/get-in-global-properties [:ctia :auth :threatgrid])
+(defn make-auth-service [get-in-config lookup-stored-identity]
```
Comment (ereteog) [inline src/ctia/auth/threatgrid.clj:73]: I would rename `lookup-stored-identity` into `lookup-identity`. This function actually ignore the store details.

**Review by ereteog — APPROVED** [review]
LGTM!

File: src/ctia/auth/threatgrid.clj (hunk 1 above)
Comment (frenchy64) [reply src/ctia/auth/threatgrid.clj:73]: I added this to the epic and I'll do this after everything is merged.

**Pull request merged**